### ISSUE TITLE
SpringBoot and Quartz Scheduler: Coercion from 'com.impossibl.postgres.system.procs.Bytes$BinDecoder$1' to 'java.sql.Blob' is not supported (com.impossibl.postgres.system.procs.Bytes$BinDecoder$1@79957f11)

### ISSUE THREAD

**Issue by archenroot** [impossibl/pgjdbc-ng#333]
I tried to migrate some basic SpringBoot and Quartz Scheduler to work with this driver and having following result:
`Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'schedulerFactoryBean' defined in class path resource [com/opencodez/configuration/ConfigureQuartz.class]: Invocation of init method failed; nested exception is org.quartz.JobPersistenceException: Couldn't retrieve trigger: Coercion from 'com.impossibl.postgres.system.procs.Bytes$BinDecoder$1' to 'java.sql.Blob' is not supported (com.impossibl.postgres.system.procs.Bytes$BinDecoder$1@79957f11) [See nested exception: java.sql.SQLException: Coercion from 'com.impossibl.postgres.system.procs.Bytes$BinDecoder$1' to 'java.sql.Blob' is not supported (com.impossibl.postgres.system.procs.Bytes$BinDecoder$1@79957f11)]
`

Actually I am having other issue with original Postgres driver, where Mysql works just fine :-)

**Comment by archenroot** [comment]
Similar/same error here: #329 

Well I encountered this error while migrating the solution from MySQL to PostgreSQL backend, now I changed original to:
```
#org.quartz.jobStore.driverDelegateClass = org.quartz.impl.jdbcjobstore.StdJDBCDelegate
org.quartz.jobStore.driverDelegateClass = org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
```

in quartz.properties, so changing STD to PostgreSQL delegate class, and the error is fixed...

I have now other error related to missing hibernate dialect, but that could be related to SpringBoot misconfig itself. Really interesting error...

**Comment by archenroot** [comment]
Just to finish my elaboration with MySQL to PostgreSQL of Quartz Scheduler running on SpringBoot project, I finally had to add hibernate.dialect into XML Spring config:
```
<bean id="sessionFactory"
		class="org.springframework.orm.hibernate4.LocalSessionFactoryBean"
		p:dataSource-ref="dataSource" >
		<property name="hibernateProperties">
			<props>
				<prop key="hibernate.cache.use_second_level_cache">false</prop>
				<prop key="hibernate.cache.use_query_cache">false</prop>
				<prop key="hibernate.show_sql">false</prop>
				<!-- HERE THE DIALECT CONFIGURATION -->
				<prop key="hibernate.dialect">org.hibernate.dialect.PostgreSQL94Dialect</prop>
			</props>
		</property>
	</bean>
```

And All works now...

**Comment by kdubb** [comment]
Fixed in #329
